source/common/runtime/runtime_impl.cc

```diff
@@ -378,8 +378,8 @@ DiskLayer::DiskLayer(absl::string_view name, const std::string& path, Api::Api&
 void DiskLayer::walkDirectory(const std::string& path, const std::string& prefix, uint32_t depth,
                               Api::Api& api) {
   ENVOY_LOG(debug, "walking directory: {}", path);
-  if (depth > MaxWalkDepth) {
```

Comment:
why this change? ThisStyle for constants is recommended in the style guide. ALL_CAPS is OK but discouraged due to possibility of clashing with macros.

Reply:
I was following the stil on lines 132-133 of `source/common/config/grpc_stream.h` and conforming the rest, but I can do it the other way around.

Reply:
+1 to ThisStyle

Reply:
updated, along with some ODR-use workarounds which I imagine was why these weren't `static constexpr` previously.

(I'd personally vote for `kThisStyle` to distinguish from types which use `ThisStyle`, but we can chat more when I get around to `StaticConstantCase` and `ConstexprVariableCase`)